packages/analytics/src/pingback.ts

```diff
@@ -27,20 +34,16 @@ const pingback = ({ gif, userId, eventType, actionType, attributes, queueEvents
         attributes,
         action_type: actionType,
         user_id: getPingbackId(),
+        analytics_response_payload: analyticsResponsePayload,
     }
 
     if (loggedInUserId) {
         newEvent.logged_in_user_id = loggedInUserId
     }
 
-    if (gif) {
-        if (!gif.analytics_response_payload) {
-            Logger.debug(`no pingback for ${gif.id}, not all endpoints have ARPs`)
-            // abort pingback, analytics_response_payload is required for gif events
-            return
-        }
-        const gifEvent = newEvent as PingbackGifEvent
-        gifEvent.analytics_response_payload = `${gif.analytics_response_payload}${
+    // add verification mode
+    if (newEvent.analytics_response_payload) {
+        newEvent.analytics_response_payload = `${newEvent.analytics_response_payload}${
             Logger.ENABLED ? '&mode=verification' : ''
```

Comment:
Maybe we can remove `&mode=verification`? Didn't we enable that on all pingbacks?

Reply:
We did, but they might disable it soon

Reply:
@dleitee cool, let's leave it then 👍